akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/Behaviors.scala

```diff
@@ -157,6 +157,22 @@ object Behaviors {
   def monitor[T](monitor: ActorRef[T], behavior: Behavior[T]): Behavior[T] =
     BehaviorImpl.intercept(new MonitorInterceptor[T](monitor))(behavior)
 
+  /**
+   * Behavior decorator that logs all messages to the [[akka.actor.typed.ActorRef]] using
```

Comment:
to the `Behavior`

because it's possible to change behavior during the life of the actor and then it might not log, which the original ActorRef is the same